test/config_test/example_configs_test.cc

```diff
@@ -8,11 +8,6 @@
 
 namespace Envoy {
 
-#if defined(__APPLE__) || defined(WIN32)
-// freebind/freebind.yaml is not supported on macOS or Windows and is disabled via Bazel.
```

Comment:
So we are testing freebind on all platforms now?

Reply:
I think it is excluded via Bazel select in the genrule that collects configs

Reply:
https://github.com/envoyproxy/envoy/blob/c8b894c558ddda6de7f1d7ffdab46216887c3db7/configs/BUILD#L30-L34

Reply:
Gotcha. Sorry for the noise.